<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1680</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ALS Terminal and Control Panel</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 16pt &quot;PibotoLt&quot;;
background-image: url(:/image/bg.jpg);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <widget class="QLabel" name="schoolName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;PibotoLt&quot;; background:transparent</string>
    </property>
    <property name="text">
     <string>Princess Chulabhorn Science High School Loei Robotics Laboratory</string>
    </property>
   </widget>
   <widget class="QLabel" name="projectName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>911</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;PibotoLt&quot;; background:transparent</string>
    </property>
    <property name="text">
     <string>Automated Librarian System Remote Server Terminal and Control Panel</string>
    </property>
   </widget>
   <widget class="QLabel" name="schoolLogo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/schoolLogo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>270</y>
      <width>581</width>
      <height>466</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="remoteLayout">
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="remotesignatureLayout">
       <item>
        <widget class="QLabel" name="remotesignatureLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Signature:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remotesignaturelineEdit"/>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <layout class="QHBoxLayout" name="remoteloginBtn">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logreqBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Request Remote Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="remotepassLayout">
       <item>
        <widget class="QLabel" name="remotepassLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remotepasslineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="remotehostLayout">
       <item>
        <widget class="QLabel" name="remotehostLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Hostname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remotehostlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="remoteIPLayout">
       <item>
        <widget class="QLabel" name="remoteIPLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>IP Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remoteIPlineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ALSremote">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string>ALS Remote Server Login</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="remoteuserLayout">
       <item>
        <widget class="QLabel" name="remoteuserLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remoteuserlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="remoteCPLayout">
       <item>
        <widget class="QLabel" name="remoteCPLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Catchphrase:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remoteCPlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="verifyLayout">
       <item>
        <widget class="QLabel" name="verifyLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Type 'RPi3' for verification [Case Sensitive]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="verifylineEdit"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="remotemacLayout">
       <item>
        <widget class="QLabel" name="remotemacLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>MAC Address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="remotemaclineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>581</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="studentLayout">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="userLayout">
       <item>
        <widget class="QLabel" name="stduserLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stduserlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="userpassLayout">
       <item>
        <widget class="QLabel" name="userpassLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="userpasslineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="salLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string>Student Account Login</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="btnLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="userloginBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="useridLayout">
       <item>
        <widget class="QLabel" name="useridLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Student ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="useridlineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>581</width>
      <height>268</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="adminLayout">
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="adminpinLayout">
       <item>
        <widget class="QLabel" name="adminpinLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>PIN Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adminpinlineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="hostLayout">
       <item>
        <widget class="QLabel" name="adminhostLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Hostname:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adminhostlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="adminloginBtn">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="adminloginButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Log In</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="adminuserLayout">
       <item>
        <widget class="QLabel" name="adminuserLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adminuserlineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="adminpassLayout">
       <item>
        <widget class="QLabel" name="adminpassLabel">
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adminpasslineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background:transparent</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="alsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background:transparent</string>
       </property>
       <property name="text">
        <string>ALS Administrator Login</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>schoolName</zorder>
   <zorder>projectName</zorder>
   <zorder>schoolLogo</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1680</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
